archetypes/pattern/additional-pages.adoc.template

```diff
@@ -0,0 +1,16 @@
+// Create a copy of this file and use it as a template for any additional pages
+---
+title: Title
+weight: 10
+---
+
+:toc:
+:imagesdir: /images
+:_content-type: ASSEMBLY
+
+// Add asciidoc modules here
```

Comment:
Can we add a suggestion about using the [include::](https://docs.asciidoctor.org/asciidoc/latest/directives/include/) directive to add modules?